src/sentry/migrations/0152_remove_slack_workspace_orgintegrations.py

```diff
@@ -21,6 +21,10 @@ def remove_slack_workspace_apps(apps, schema_editor):
         Integration.objects.filter(provider="slack")
     ):
         if not integration.metadata.get("installation_type"):
+            # classic bot apps use the `user_access_token` but
+            # may be missing the installation_type. Don't delete these
+            if "user_access_token" in integration.metadata:
```

Comment:
Maybe also update the `installation_type` for them?

Reply:
hmmm @manuzope do you have thoughts here? about adding a "classic_bot" type?

Reply:
@BYK im leaning towards not updating because we only added that to be able to distinguish between the new bot and the workspace apps. I think the `"user_access_token"` is still the best indicator that something is a classic bot, because that's actually used in the code

Reply:
Wfm @MeredithAnya, you're the expert.